src/test/kotlin/com/pubnub/api/subscribe/eventengine/worker/TransitionFromHandshakeFailedStateTest.kt

```diff
@@ -60,10 +60,10 @@ class TransitionFromHandshakeFailedStateTest {
 
         // then
         assertEquals(
-            SubscribeState.Receiving(channels, channelGroups, subscriptionCursor), state
+            SubscribeState.Handshaking(channels, channelGroups, subscriptionCursor), state
```

Comment:
test name probably should be changed as well to:
can_transit_from_HANDSHAKE_FAILED_to_HANDSHING_when_there_is_SUBSCRIPTION_RESTORED_even